utils/docify/README.md

```diff
@@ -0,0 +1,31 @@
+Docify is a utility that can insert docstrings into Python type stub files.
+We use this because we sometimes want to put docstrings for functions into
+the stubs, because it is hard to extract them from source. This is primarily
+used for some pandas APIs for now.
+
+To install:
+
+    PBR_VERSION=1.0.0 pip install .
```

Comment:
Is this a command? Or an environment var that needs to be set followed by a command?

Reply:
The latter. I am using PBR and it needs that variable.

Reply:
Ok, this seems confusing then since "PBR_VERSION=1.0.0 pip install ." as a single command line command is not valid. Can you turn this into multiple commands/lines that would be valid? Maybe this?
```suggestion
    set PBR_VERSION=1.0.0
    pip install .
```

Reply:
I am going to disagree with you on this. In your approach, the env var now has a scope beyond the command which is the only place it is used, and it is no longer obvious why it is needed. It is a common paradigm in *nix shells to do it the way I have; the scope is just for that command and so it is better bounded and also clear that it is needed for that command.

Reply:
That's fair. Then you should be clear that that install command is meant to be run in a particular shell or on a particular OS, because as written it won't work in Windows cmd.

Reply:
Btw, who is expected to run this script? People on the Pylance team? External folks?